readme-vars.yml

```
@@ -43,6 +43,7 @@ optional_block_1: false
 app_setup_block_enabled: true
 app_setup_block: |
   Edit the `ddclient.conf` file found in your `/config` volume (also see official [ddclient documentation](https://ddclient.net)). This config file has many providers to choose from and you basically just have to uncomment your provider and add username/password where requested. If you modify ddclient.conf, ddclient will automaticcaly restart and read the config.
+  Use the template from `/defaults/ddclient.conf` in order to e.g. keep the `daemon` interval configuration, as otherwise `ddclient` uses an re-check interval of one second, which is too much for many dynamic DNS services.
```

So what about this then?  @Roxedus 

```suggestion
  The default settings from `/defaults/ddclient.conf` can be copied to e.g. keep the `daemon` interval configuration, as otherwise `ddclient` uses an re-check interval of one second, which is too much for many dynamic DNS services.
```

The thing that `/config/ddclient.conf` is the configuration file is already explained in the paragraph there, so I did not mention it a second time as that feels unnecessary IMHO.

---

The file is already copied to the mentioned path, rewording it to something like "setting X sets the timeout to Y, this may be beneficial because Z"

---

Well… it may be copied thee, but if you mount your own file, it of course overlays the default thing there, that is wnat this PR is about :sweat_smile: 

Alternatively, if you wish:
```suggestion
  The default settings `/defaults/ddclient.conf` can be used when you write your own configuration file. It is especially useful to set the `daemon` interval configuration as suggested there, because if unset `ddclient` uses an re-check interval of one second, which is too much for many dynamic DNS services.
```

---

No, we don't want people to touch the default file, as I mentioned earlier.

---

This is not what I try to imply with this doc change.

Lets go one step back, so please have a look at the PR description at "Benefits of this PR and context:" and [also this forum thread](https://talk.desec.io/t/running-into-429-too-many-requests-for-https-checkipv4-dedyn-io/284).
The point is, you obviously want to provide your own config file, because otherwise you cannot configure ddclient.

Now, if you do that though, ddclient ignores the `/defaults/ddclient.conf` file, as it seems. That's why I thought let's add that change to the config that you need to configure `daemon`, because otherwise you may run into rate limits.
That's all this is about…

Maybe I do understand something wrong here though or it is supposed to use the `/defaults/ddclient.conf` file as a default, but in my tests/case it does not. So let me know if I understand something wrong here…

---

To clarify: Of course I never overwrote that file and always used `/config` (IIRC) as the Readme recommends. It anyway makes ddclient ignore the default file, which seems to be the core issue here…

---

We just use the default config file from ddclient, so it would be better if you asked ddclient to change the default Daemon value. 
This only happens when using a specific page to check the IP, right?

---

It's a thing to try, at least, okay, so opened https://github.com/ddclient/ddclient/issues/352